tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_binary_divtrunc.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import pytest
+import tt_lib
+from tests.tt_eager.python_api_testing.unit_testing.backward_ops.utility_funcs import compare_results, data_gen_pt_tt
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+def test_bw_binary_divtrunc(input_shapes, device):
+    grad_data, grad_tensor = data_gen_pt_tt(input_shapes, device)
+    in_data = torch.Tensor(size=input_shapes).uniform_()
+    in_data.requires_grad = True
+    input_tensor = (
+        tt_lib.tensor.Tensor(in_data, tt_lib.tensor.DataType.BFLOAT16).to(tt_lib.tensor.Layout.TILE).to(device)
+    )
+    other_data, other_tensor = data_gen_pt_tt(input_shapes, device, True)
+
+    pyt_y = torch.div(in_data, other_data, rounding_mode="trunc")
+
+    tt_output_tensor_on_device = tt_lib.tensor.binary_divtrunc_bw(grad_tensor, input_tensor, other_tensor)
```

Comment:
Why are we calling it binary_divtrunc_bw ?     Ideally we would follow the torch api and pass round_mode="trunc".   What's the right thing to do here?

Reply:
Yeah we can have like same as torch api